plugins/woocommerce/src/Internal/RestApi/Routes/V4/Settings/PaymentGateways/Schema/AbstractPaymentGatewaySettingsSchema.php

```
@@ -0,0 +1,649 @@
+		return $settings;
+	}
+
+	/**
+	 * Get the item response.
+	 *
+	 * @param WC_Payment_Gateway $gateway Payment gateway object.
+	 * @param WP_REST_Request    $request Request object.
+	 * @param array              $include_fields Fields to include in the response.
+	 * @return array The item response.
+	 */
+	public function get_item_response( $gateway, WP_REST_Request $request, array $include_fields = array() ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)